### PR TITLE
GTM and Gtag fixes

### PR DESCRIPTION
**Pull request by joshocalico**

### CODE REVIEW

File: automation/fluidity/common-mainnet.yml [hunk 1]
```diff
@@ -7,3 +7,6 @@ SECRET_FLU_SENTRY_URL: /fluidity/sentryUrl
 SECRET_FLU_FRONTEND_SENTRY_URL: /fluidity/frontend/sentryURL
 SECRET_FLU_NPM_GITHUB_TOKEN: /fluidity/npm/github_token
 SECRET_FLU_MORALIS_TOKEN: /fluidity/moralis/token
+
+ENV_GTM_ID: GTM-W7QJGR2
```
Comment (af-afk) [inline automation/fluidity/common-mainnet.yml]: Isn't this set and available in mainnet.yml for app/website already?

File: web/app.fluidity.money/Dockerfile.frontend [hunk 2]
```diff
@@ -18,6 +18,9 @@ RUN make frontend
 
 FROM fluidity/runtime-web-container:latest
 
+ENV GTAG_ID ${GTAG_ID}
```
Comment (af-afk) [inline web/app.fluidity.money/Dockerfile.frontend]: Prepend it with FLU_

File: web/app.fluidity.money/app/constants.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Not confident that this will work properly in the interim. So I'm providing a default.
+export const GTM_ID = process?.env["GTM_ID"] ?? "GTM-W7QJGR2";
```
Comment (af-afk) [inline web/app.fluidity.money/app/constants.ts]: I don't like the idea of hardcoding it so it's always on, if it's not available the feature should just not enable

File: web/app.fluidity.money/app/root.tsx [hunk 4]
```diff
@@ -22,6 +22,8 @@ import CacheProvider from "contexts/CacheProvider";
 import { useEffect, useState } from "react";
 import { CookieConsent } from "@fluidity-money/surfing";
 
+import { GTM_ID, GTAG_ID } from "./constants";
```
Comment (af-afk) [inline web/app.fluidity.money/app/root.tsx]: Why have a constants file in the first place?

File: web/fluidity.money/src/utils/gtag.ts [hunk 5]
```diff
@@ -1,10 +1,15 @@
-export const GA_DEST_ID = 'G-EF68MNJRJ7';
-export const GA_TRACKING_ID = 'GT-NM2HFZW';
+// export const GA_DEST_ID = 'G-EF68MNJRJ7';
+// export const GA_TRACKING_ID = 'GT-NM2HFZW';
+
+export const GTAG_ID = 
+  process.env["GTAG_ID"] ?? "G-EF68MNJRJ7";
+export const GTM_ID = 
+  process.env["GTM_ID"] ?? "GTM-W7QJGR2";
```
Comment (af-afk) [inline web/fluidity.money/src/utils/gtag.ts]: Then this is set, though different from the previous constants approach

**Review by af-afk — DISMISSED** [review]

**Review by glo12345 — APPROVED** [review]

File: automation/fluidity/common-mainnet.yml (hunk 1 above)
Comment (joshocalico) [reply automation/fluidity/common-mainnet.yml:11]: Only for the app unfortunately. I've just altered it so it's in the "root" common file. 
At least, I'm assuming that's how the fluidity cli code works granted there are other ENV entries in there.

Comment (joshocalico) [reply automation/fluidity/common-mainnet.yml]: Only for the app unfortunately. I've just altered it so it's in the "root" common file.
At least, I'm assuming that's how the fluidity cli code works granted there are other ENV entries in there.

**Pull request merged**